ts/src/connections/index.ts

```diff
@@ -18,3 +18,11 @@ export * from './tunnelRelayStreamFactory';
 export * from './defaultTunnelRelayStreamFactory';
 export * from './tunnelRelayTunnelClient';
 export * from './tunnelRelayTunnelHost';
+export * from './tunnelConnection';
+export * from './tunnelConnectionBase';
+export * from './tunnelSession';
+export * from './connectionStatus';
```

Comment:
Does the `TunnelSession` class need to be public?

Reply:
It has methods on it that look like they should be internal implementation details, not callable by applications.